### PR TITLE
docs: Add a note about the wip extension status

### PR DESCRIPTION
**Pull request by abeyad**
Signed-off-by: Ali Beyad <abeyad@google.com>

### CODE REVIEW

**Review by alyssawilk — DISMISSED** [review]
LGTM but I'd be inclined to throw this by Harvey or Adi

**Comment by alyssawilk** [comment]
cc @htuch @yanavlasov

**Comment by alyssawilk** [comment]
oh and fix_format apparently :-/

File: api/STYLE.md [hunk 1]
```diff
@@ -34,6 +34,9 @@ In addition, the following conventions should be followed:
   implementation. These indicate that the entity is not implemented in Envoy and the entity
   should be hidden from the Envoy documentation.
 
+* For extensions that are a work-in-progress or the documentation is hidden, set its `status`
```
Comment (adisuissa) [inline api/STYLE.md:37]: RE "documentation is hidden", I assume you mean the entire documentation is hidden (and not some field or sub-message).

**Review by adisuissa — COMMENTED** [review]
LGTM, I'll let @htuch chime in.

File: api/STYLE.md (hunk 1 above)
Comment (abeyad) [reply api/STYLE.md:37]: I noticed it for when the proto config has the "[#not-implemented-hide:]" annotation in the comments.  The proto also has the "[#extension: ...]" annotation, which I assume is where it gets correlated to the extension entry in `extensions_metadata.yaml`

File: api/STYLE.md [hunk 2]
```diff
@@ -34,6 +34,9 @@ In addition, the following conventions should be followed:
   implementation. These indicate that the entity is not implemented in Envoy and the entity
   should be hidden from the Envoy documentation.
 
+* For extensions that are a work-in-progress or the config proto documentation is hidden with
+  `[#not-implemented-hide:]`, set its `status` field to `wip` in `extensions_metadata.yaml`.
```
Comment (htuch) [inline api/STYLE.md:38]: LGTM as general guidance.

**Review by alyssawilk — APPROVED** [review]

**Pull request merged**